tools/stacktrace.toit

```diff
@@ -10,21 +10,26 @@ usage:
 
 OBJDUMP ::= "xtensa-esp32-elf-objdump"
 
+ELF_FILE ::= "path/to/toit.elf"
+
 main args/List:
-  parser := arguments.ArgumentParser
-  parser.describe_rest ["/path/to/toit.elf"]
-  parser.add_flag "disassemble" --short="d"
-  parser.add_option "objdump" --default=OBJDUMP
-  parser.add_option "backtrace" --default="-"
-  parsed := parser.parse args:
-    usage
-    exit 1
-  if args.size < 1: usage
+  parsed := null
+  parser := cli.Command "stacktrace"
```

Comment:
Add the old 'usage' as `--long_help`?
The second example could also just be an example:
```
--examples=[
  cli.Example "Make a backtrace for a 'build/toit.elf' file readable"
      --arguments="backtrace=\"Backtrace:0x400870c0:0x3ffc9df0 0x4010661d:0x3ffc9e70 0x401143a3:0x3ffc9ea0\" build/toit.elf"
]
```